Resources/Prototypes/game_presets.yml

```
@@ -310,4 +310,18 @@
     - BasicStationEventScheduler
     - BasicRoundstartVariation
     - GameRuleMeteorScheduler
+
+- type: gamePreset
+  id: SecretCult #For Admin Use: Runs Cult, but shows "Secret" in lobby.
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)